package.json

```diff
@@ -148,6 +148,7 @@
     "build:copy-files": "node scripts/copyBuildFiles.js",
     "build:commonjs": "cross-env NODE_ENV=production babel ./src/components --out-dir ./lib --ignore index.js",
     "publish": "./scripts/publish.sh",
+    "publish-beta": "node ./scripts/publish-beta.js",
```

Comment:
Hey @msanroman, would it make sense to rename this script to just beta instead of publish-beta, because its not too far away from accidentally typing publish 😅 especially for people using autocomplete in their console?

Reply:
Good one 😆 What about `beta:publish`? Would love to be still clear this is publishing a beta version 😄

Reply:
that sounds perfect 🙌